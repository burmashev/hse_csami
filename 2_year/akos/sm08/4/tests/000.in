1 15
-1000 100
833.14 -0.0000
