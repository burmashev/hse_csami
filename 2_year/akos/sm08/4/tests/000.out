0.06666667
-10.00000000
ERROR
